cabal-install/tests/IntegrationTests2.hs

```diff
@@ -113,22 +118,42 @@ main = do
   defaultMainWithIngredients
     (defaultIngredients ++ [includingOptions projectConfigOptionDescriptions])
     ( withProjectConfig $ \config ->
-        testGroup
+        sequentialTestGroup
```

Comment:
`tasty` story for sequential execution is admittedly finicky. If we want to run entire test suite in a single thread, I'd rather slap `defaultMain $ localOption (NumThreads 1) $ ...` on the very top level.

Reply:
I will add that as well as a fail-safe but it does seem more declarative to specify that the tests have to run sequentially.